config/jobs/kubernetes/sig-gcp/gpu/README.md

```diff
@@ -0,0 +1,95 @@
+GPU Job Scheduling
+======
+
+A finite number of GPU projects is available (at time of writing, seventeen). If all the jobs
+are left to run on an interval without further intervention, we will exhaust the available projects
+and jobs will fail trying to get one.
+
+This file sets out an explicit schedule to minimise the number of concurrent jobs we've used.
+It's probably going to go stale, but is accurate at time of writing.
+
+Currently we only need to use six projects, leaving eleven for PR jobs and other exceptional events. 
+
+Artisanal
+------
+
+### 2 hours
+a) `ci-kubernetes-e2e-gke-device-plugin-gpu` (20m) (00:00)  
+b) `ci-kubernetes-e2e-gke-device-plugin-gpu-beta` (20m) (00:30)  
+c) `ci-kubernetes-e2e-gke-device-plugin-gpu-ubuntu` (20m) (01:00)  
+d) `ci-kubernetes-e2e-gce-device-plugin-gpu` (22m) (01:30)  
+e) `ci-kubernetes-e2e-gce-device-plugin-gpu-beta` (22m) (00:00)
+
+### 3 hours
+f) `ci-kubernetes-e2e-gke-device-plugin-gpu-monitoring` (21m) (00:30)  
+g) `ci-cri-containerd-e2e-gce-device-plugin-gpu` (22m) (01:30)
+
+### 4 hours
+h) `ci-kubernetes-e2e-gci-gke-autoscaling-gpu-k80` (1h33m) (00:00)  
+i) `ci-kubernetes-e2e-gci-gke-autoscaling-gpu-p100` (1h33m) (02:00)
+
+### 6 hours
+j) `ci-kubernetes-e2e-gke-device-plugin-gpu-stable1` (21m) (01:00)  
+k) `ci-kubernetes-e2e-gce-device-plugin-gpu-stable1` (23m) (03:00)  
+l) `ci-kubernetes-e2e-gce-gpu-stable2` (20m, always fails) (05:00)  
```

Comment:
I think we can remove this old test now. The feature it tested (KUBE_FEATURE_GATES=Accelerators=true) was already removed from k8s tree since 1.9.